tests/application.rs

```diff
@@ -1,10 +1,11 @@
 use mac_notification_sys::*;
 
 #[test]
-#[should_panic]
 fn set_application_again() {
-    let _ = set_application("com.apple.Terminal").unwrap();
-    let _ = set_application("com.apple.Terminal").unwrap();
+    let mut result = set_application("com.apple.Terminal");
+    assert!(result.is_ok());
+    result = set_application("com.apple.Terminal");
+    assert!(result.is_err());
```

Comment:
nitpick: unwrapping this first `result` actually preferred - generally nicer debug experience should it actually be an `Err`

Reply:
Good point!